reference/docs-conceptual/developer/cmdlet/how-to-invoke-a-cmdlet-from-within-a-cmdlet.md

```diff
@@ -6,14 +6,15 @@ title: How to Invoke a Cmdlet from Within a Cmdlet
 ---
 # How to Invoke a Cmdlet from Within a Cmdlet
 
-This example shows how to invoke a cmdlet from within another cmdlet, which allows you to add the
-functionality of the invoked cmdlet to the cmdlet you are developing. In this example, the
-`Get-Process` cmdlet is invoked to get the processes that are running on the local computer. The
-call to the `Get-Process` cmdlet is equivalent to the following command. This command retrieves all
-the processes whose names start with the characters "a" through "t".
+This example shows how to invoke a binary cmdlet that derives from `Cmdlet` directly from within
+another binary cmdlet, which allows you to add the functionality of the invoked cmdlet to the binary
+cmdlet you are developing. In this example, the `Get-Process` cmdlet is invoked to get the processes
+that are running on the local computer. The call to the `Get-Process` cmdlet is equivalent to the
+following command. This command retrieves all the processes whose names start with the characters
+"a" through "t".
```

Comment:
The meaning of the word `cmdlet` was changed in the last couple of years to include advanced functions/scripts. I've updated the wording to account for this (though the first sentence is approaching (or has arrived at) a run-on for sure).

Also the class that the *caller* inherits isn't actually important, only the class of the binary cmdlet to be invoked.